common/src/main/java/io/druid/concurrent/Execs.java

```diff
@@ -63,6 +83,15 @@ public static ThreadFactory makeThreadFactory(String nameFormat)
    */
   public static ExecutorService newBlockingSingleThreaded(final String nameFormat, final int capacity)
   {
+    return newBlockingSingleThreaded(nameFormat, capacity, Thread.NORM_PRIORITY);
+  }
+
+  public static ExecutorService newBlockingSingleThreaded(
+      final String nameFormat,
+      final int capacity,
+      final Integer priority
```

Comment:
Yes, this is actually the key one.

https://github.com/druid-io/druid/pull/984/files#diff-e7c5ce8efa97c42949b01290c2bb4a47R530